crates/lib/src/lib.rs

```diff
@@ -8,7 +8,7 @@
 #![forbid(unsafe_code)]
 #![warn(future_incompatible)]
 #![warn(rust_2018_compatibility, rust_2018_idioms)]
-#![deny(
+#![warn(
```

Comment:
Across the board, we should get out of the habit of causing compiler errors for style issues, they just add noise while development is in full swing. They can be raised as errors during linting.